src/main.py

```diff
@@ -2716,6 +2717,15 @@ def post_all(self, o):
         email = post["email"]
         signature = post["signature"]
         quicklinks = post["quicklinksid"]
+        rawfavouritereports = post["favouritereportsid"]
+        favouritereports = ""
```

Comment:
This is for the string that we're going to store in the config table, I'd called it quickreportscfg. I would make it a list instead, that you append to in your loop so we can use join to add the commas. quickreportscfg = []

Reply:
:) I took that approach originally but couldn't get the join to work and was keen to move on.

Reply:
I renamed quickreportscfg and thought while I was renaming fields I'd change all the variable names I'd created that included Favourite to Quick. I found some other bugs (possibly ones you've put further down this list) that I don't think were related to me renaming the variables. Anyway, sorted out the bugs, renamed the fields and made an array and got the join working, I think I was confusing javascript joins with python joins :)

Reply:
Yeah, they're the opposite way round. Javascript join is a method of arrays, so you'd do [ ].join(",") - but Python join is a method of String, so you do ",".join([])